ee/codegen/src/generators/sandbox-file.ts

```diff
@@ -0,0 +1,113 @@
+import { python } from "@fern-api/python-ast";
+import { AstNode } from "@fern-api/python-ast/core/AstNode";
+
+import { BasePersistedFile } from "./base-persisted-file";
+
+import type { Inputs } from "./inputs";
+import type { WorkflowFile } from "./workflow";
+
+export declare namespace SandboxFile {
+  interface Args extends BasePersistedFile.Args {
+    workflowFile: WorkflowFile;
+    inputsFile: Inputs;
+    sandboxInputs: Record<string, unknown>[];
+  }
+}
+
+export class SandboxFile extends BasePersistedFile {
+  private readonly workflowFile: WorkflowFile;
+  private readonly inputsFile: Inputs;
+  private readonly sandboxInputs: Record<string, unknown>[];
+
+  public constructor({
+    workflowContext,
+    workflowFile,
+    inputsFile,
```

Comment:
[blcoking] I'd really really like to avoid passing file ast nodes into another ast node. This makes it impossible to generate files in parallel and starts introducing dependencies between them. 

We should use/extend contexts as needed to achieve the same thing.